lib/awful/layout/init.lua

```diff
@@ -162,6 +162,11 @@ function layout.arrange(screen)
     delayed_arrange[screen] = true
 
     timer.delayed_call(function()
+        if not screen.valid then
```

Comment:
My config say otherwise, I need the `pcall` anyway or I get an assert "attempt to call an invalid object" in `if c.valid then`. At least, that was still the case not so long ago.